adapted-articles/J/JesusChristTeachingsof.xml

```diff
@@ -14,77 +14,77 @@
-<p>In the Sermon on the Mount, Jesus assured his followers that their heavenly Father knows about their needs (<a href="?bref=Matt.6.32">Mt 6:32</a>; <a href="?bref=Luke.12.30">Lk 12:30</a>), on the strength of which they are exhorted not to be anxious. This gives some insight into the way in which Jesus’ teachings about God has a bearing on practical issues.</p>
+<p class="fl">Anyone who comes to the teachings of Jesus after reading the OT will at once recognize that much of his teaching about God is the same. Since Jesus, as all orthodox Jews of his day, accepted the testimony of the OT as inspired, it is not surprising that his approach to God was similar. This is especially true of his assumption that God was Creator. He taught a special divine care over the created order and affirmed that God watched over such small creatures as the sparrow (<a href="?bref=Matt.10.29">Mt 10:29</a>). There is no support in the teachings of Jesus for the view that God is uninterested in the world he made.</p>
+<p>One of the most characteristic titles Jesus used for God was Father. This was not new, because the idea occurs in the OT, where God is seen as Father of his people Israel. This kind of fatherhood was national rather than personal. In the period between the old and new testaments the Jews came to regard God as so holy that he was removed from direct contact with human affairs. There had to be mediators between God and people. This elevated notion of God did not lead to the idea of God as Father. It is against this background that we must see Jesus’s unique teaching of the personal fatherhood of God. There is some evidence in Judaism that prayer to God as “Our Father” was known, but what distinguishes Jesus’s teachings from that of his contemporaries is that the fatherhood of God was central to what he taught.</p>
+<p>The father-son relationship is particularly vivid in John’s Gospel, where Jesus as Son is seen to be in close communion with God as Father. This comes out strongly in Jesus’s prayer in <a href="?bref=John.17.1-26">John 17</a> and in the frequent assertions that the Father had sent the Son and that the Son was doing the will of the Father. It is this strong relationship between God and Jesus in terms of fatherhood and sonship that led Jesus to teach men to approach God in the same way. The Lord’s Prayer at once recognizes this in its opening words. It is particularly important to note that “Our Father” precedes “may your name be kept holy” (<a href="?bref=Matt.6.9">Mt 6:9</a>), because the it is a more intimate idea and prepares the way for the more remote. Jesus never taught men to approach God with terror.</p>
```

Comment:
"because the it is a more intimate idea" ... the "the" has to go. The original sentence is terrible, maybe "We should note that the intimacy of "Our Father" comes before "may your name be kept holy", ie the intimate relationship sets the context for a God who is holy and "other".

Reply:
How about:
> It is particularly important to note that the intimacy of “Our Father” comes before “may your name be kept holy” (Mt 6:9). Jesus never taught men to approach God with terror.

That way we keep the author’s original superlativeness, and also keep it simple(r).

Reply:
ok